chainer/_runtime_info.py

```diff
@@ -0,0 +1,36 @@
+import sys
+
+import numpy
+import six
+
+import chainer
+from chainer.backends import cuda
+
+
+class _RuntimeInfo(object):
```

Comment:
I'm not sure when to prefix `_` to class names in Chainer. Any thoughts?

Reply:
I wanted to signify that this class is implementation detail and not supporsed to instantiated by user.
(I think there's no established convention in the world when or when not to add this prefix, but I simply thought it won't harm anyone)